mobile/src/components/course-module/components/task/task.tsx

```diff
@@ -105,7 +66,7 @@ const Task: FC = () => {
     if (user && moduleId) {
       dispatch(
         courseModulesActions.getTask({
-          menteeId: user.id,
+          menteeId: menteeId as number,
```

Comment:
```suggestion
          menteeId: menteeId,
```

Reply:
![image](https://user-images.githubusercontent.com/82529236/192018869-854307e2-8a5d-4c2d-bcb7-fdfec726f06a.png)

Reply:
And also change in dependencies